src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -74,15 +74,15 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
                 Slots.RANK_PARAM_IS_DUPLICATES_ALLOWED, Const.ParamsNames.FEEDBACK_QUESTION_RANKISDUPLICATESALLOWED,
                 Slots.RANK_ARE_DUPLICATES_ALLOWED_VALUE, Boolean.toString(isAreDuplicatesAllowed()),
                 Slots.RANK_IS_MAX_OPTIONS_TO_BE_RANKED_ENABLED, isMaxOptionsToBeRankedEnabled ? "" : "disabled",
-                Slots.RANK_DISPLAY_MAX_OPTIONS_HINT, isMaxOptionsToBeRankedEnabled ? "" : "hidden",
+                Slots.RANK_DISPLAY_MAX_OPTIONS_HINT, "hidden",
```

Comment:
If this is no longer a variable, why not hard-code `Slots.RANK_DISPLAY_MAX_OPTIONS_HINT` in the template itself?

Reply:
Rank recipients and Rank options share the same submission page template, for Rank options we show the hint, and for Rank recipients, we hide it and show instructions instead. Also, the flexibility of using `hidden` to hide the hints is useful in cases where both min/max are disabled or either of them is disabled, in which case we hide whatever is disabled. So, `hidden` is pretty useful here commonly used by Rank options and recipients.